conans/test/unittests/client/generators/cmake_test.py

```diff
@@ -51,10 +52,12 @@ def variables_setup_test(self):
         conanfile.initialize(Settings({}), EnvValues())
         ref = ConanFileReference.loads("MyPkg/0.1@lasote/stables")
         cpp_info = CppInfo("dummy_root_folder1")
+        cpp_info.name = ref.name
```

Comment:
What happens if you don't assign this?

Reply:
When `cpp_info.name` is used inside the generator, the None type makes it crash

`AttributeError: 'NoneType' object has no attribute 'upper'`